app/controllers/verify/sessions_controller.rb

```diff
@@ -16,44 +16,39 @@ def new
     end
 
     def create
-      process_step
-    end
-
-    private
+      result = step.submit
+      analytics.track_event(Analytics::IDV_BASIC_INFO_SUBMITTED, result.to_h)
```

Comment:
We do. This PR is only a refactoring. No changes to functionality, so you wouldn't see any controller specs added. You'd need to see the entire spec file.